### PR TITLE
i10–convert function comments to proper jsDoc

### PR DESCRIPTION
**Pull request by eshwaribhide**
–This has been done for all files where necessary.
–Other things that were changed included some spacing inconsistencies, along with tense for purpose statements (very minor).
–I also was bothered by a typo in page-content/Glossary, where "sequence" was misspelled, so that has also been changed.

### CODE REVIEW

File: server-code/Main.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ var scriptProps = PropertiesService.getScriptProperties(); // Google apps script
 /**
  * Serves up the website HTML from Index.html
  * 
- * @return {HtmlOutput} request 
+ * @return {HtmlOutput} 
  */
 function doGet(request) {
```
Comment (eshwaribhide) [inline server-code/Main.js:13]: this parameter is never used btw

Comment (jamescd18) [reply server-code/Main.js:13]: Yes, it never is, but I believe it is a required parameter.

File: client-code/JavaScript.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
      * Handle errors and failures when executing async server calls.
      * Adds a dismissable Boostrap error alert to the top of the page with the error message, and scrolls the window to the top of the page.
      * 
-     * @param { Object } Error object returned by the server
+     * @param {Object} Error object returned by the server
```
Comment (kevinyu328) [inline client-code/JavaScript.html:6]:
```suggestion
     * @param {Object} error - The error object returned by the server.

```

File: client-code/JavaScript.html [hunk 3]
```diff
@@ -21,8 +21,8 @@
      * Takes the HTML payload object returned by the server and places it within the specified HTML DOM object specified when executing the server call.
      * Also scrolls the window to the top of the page to ensure users see relevant content and / or success messages.
      * 
-     * @param { String } HTML string content payload returned from the server
-     * @param { Object } HTML DOM Object to provide the given HTML payload
+     * @param {String} HTML string content payload returned from the server
+     * @param {Object} HTML DOM Object to provide the given HTML payload
```
Comment (kevinyu328) [inline client-code/JavaScript.html:25]:
```suggestion
     * @param {String} htmlPayload - HTML string content payload returned from the server
     * @param {Object} htmlObject - HTML DOM Object to provide the given HTML payload

```

File: client-code/JavaScript.html [hunk 4]
```diff
@@ -92,8 +92,12 @@
 
     /* Changes page functions */
 
-    // onSuccessHTMLStatusUpdateSubmit : HTML Object[HTML] -> n/a
-    // success handler for async server calls, inserts returned payload into provided element's HTML
+    /**
+     * Success handler for async server calls, inserts returned payload into provided element's HTML
+     * 
+     * @param {String} HTML string content payload returned from the server
+     * @param {Object} HTML DOM Object to provide the given HTML payload
```
Comment (kevinyu328) [inline client-code/JavaScript.html:99]:
```suggestion
     * @param {String} htmlPayload - HTML string content payload returned from the server
     * @param {Object} htmlObject - HTML DOM Object to provide the given HTML payload

```

File: server-code/ChangeForm.js [hunk 5]
```diff
@@ -47,7 +46,7 @@ function getNextChangeRequestId(requestType) {
 }
 
 /**
-* Convert long-form string to single character short-form of change request type
+* Converts long-form string to single character short-form of change request type
 *
 * @param {String} - The long form change request type
 * @return {String} - Short form change request type
```
Comment (kevinyu328) [inline server-code/ChangeForm.js:52]:
```suggestion
* @param {String} requestType - The long form change request type
* @return {String}  Short form change request type
```

File: server-code/Gantt.js [hunk 6]
```diff
@@ -6,9 +6,9 @@
  * Provides HTML content for viewing the Gantt chart in a small viewer or a full-page view 
  * built using a modal and sized with the provided width and height for the page
  * 
- * @param { number } width HTML page widht
- * @param { number } height HTML page height
- * @return { string } Formatted HTML string content containing iframes for the Gantt chart
+ * @param {number} width HTML page widht
+ * @param {number} height HTML page height
+ * @return {string} Formatted HTML string content containing iframes for the Gantt chart
```
Comment (kevinyu328) [inline server-code/Gantt.js:11]:
```suggestion
 * @param {number} width - HTML page width
 * @param {number} height - HTML page height
 * @return {string} Formatted HTML string content containing iframes for the Gantt chart
```

File: server-code/Changes.js [hunk 7]
```diff
@@ -113,8 +128,12 @@ function getNewStatusInput() {
     return html;
 }
 
-// setNewStatus : Object[New Status] -> HTML
-// Sets status of WBS # in newStatus to number in newStatus, return HTML for success message
+/**
+ * Sets status of WBS # in newStatus to number in newStatus
+ * 
+ * @param {Object} newStatus 
+ * @return {String} HTML for success message
```
Comment (kevinyu328) [inline server-code/Changes.js:135]:
```suggestion
 * @param {Object} newStatus - the new status for the WBS# to be set to. 
 * @return {String} An HTML alert success message.
```

**Review by kevinyu328 — CHANGES_REQUESTED** [review]
Great work converting the comments to JSDoc! However, some of the parameter descriptions and return statements are not filled in. I did a couple to help you get started. I know there are quite a few comments to fill in and they can get quite tedious, so don't feel pressured to have to do them all at once. Feel free to work on a couple a day and finish it that way. Here are some [examples of JSDoc](https://jsdoc.app/howto-es2015-classes.html) that you can refer to when working on the comments. Please make sure the formatting of the comments are consistent. Thanks!

File: server-code/Main.js (hunk 1 above)
Comment (eshwaribhide) [reply server-code/Main.js:13]: Yes, it's just me not knowing anything about web dev.

**Review by kevinyu328 — APPROVED** [review]

**Comment by kevinyu328** [comment]
Great work! Thanks for adding all the descriptions!

**Pull request merged**